src/main/java/org/example/OccurrenceWordPoc.java

```diff
@@ -0,0 +1,55 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class OccurrenceWordPoc {
+    /*
+    Entering the path
+    once if the input is received it shows processing
+    and executes if the file is present or not present
+     */
+    public static void main(String[] args) {
+        if (args.length == 2) {
+            String inputFilePath = args[0];
+            String searchWord = args[1];
+            System.out.println(inputFilePath);
+            File inputFile = new File(inputFilePath);
+            System.out.println("Processing......");
+            if (inputFile.exists()) {
+                System.out.println("File already exists");
+            } else {
+                System.out.println("File dont exist");
+                return;
+            }
+            performRead(inputFilePath, searchWord);
+        }
```

Comment:
You need an else condition and show an error message to the user

Reply:
Updated